examples/webservice/java/src/main/java/es/nangel/logstash/lookup/webservice/test/TestController.java

```diff
@@ -0,0 +1,18 @@
+package es.nangel.logstash.lookup.webservice.test;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class TestController {
+
+    @RequestMapping(value = "/json", produces = "application/json")
+    public Map<String, String> getElementJSON() {
+        Map<String, String> toReturn = new HashMap<>();
```

Comment:
Hello! that test is only to have a client.
But you are right! that test has no sense. I will clean the project asap!